tests/trading/test_0x_adapter.py

```diff
@@ -0,0 +1,56 @@
+    trade = [
+        TRADE_TYPE["EXACT_IN_SINGLE"], 
+        env.tokens["COMP"], 
+        env.tokens["WETH"], 
+        amount, 
+        0, 
+        chain.time(), 
+        tradeData
+    ]
+    # Vault does not have permission to sell wstETH
+    with brownie.reverts():
+        env.tradingModule.executeTrade.call(DEX_ID["ZERO_EX"], trade, {"from": accounts[0]})
```

Comment:
Can you add another test to confirm that the wstETH/ETH vault also does not have permissions to sell via ZERO_EX